app/views/main.scala.html

```diff
@@ -69,7 +69,7 @@
 }
 
 @isVerify = @{
-    authProvider.contains("IDA")
+    authProvider.contains(TaiConstants.AuthProviderVerify)
```

Comment:
@vimalsuresh need to check the correct menu bar is shown but I believe this is under test via ATs

Reply:
Have AT's to test menu links, looks good